examples/daal4py/pca.py

```diff
@@ -19,23 +19,12 @@
 from pathlib import Path
 
 import numpy as np
+from readcsv import pd_read_csv
 
 import daal4py as d4p
 
-# let's try to use pandas' fast csv reader
-try:
-    import pandas
 
-    def read_csv(f, c=None, t=np.float64):
-        return pandas.read_csv(f, usecols=c, delimiter=",", header=None, dtype=t)
-
-except ImportError:
-    # fall back to numpy loadtxt
-    def read_csv(f, c=None, t=np.float64):
-        return np.loadtxt(f, usecols=c, delimiter=",", ndmin=2)
-
-
-def main(readcsv=read_csv, method="svdDense"):
+def main(readcsv=pd_read_csv, method="svdDense"):
```

Comment:
remove method here? assuming the removals are when its unused in the example

Reply:
No I am removing duplicated code. The CSV read helpers are now defined in a single place `examples/utils/readcsv.py`

Reply:
I mean removing the method="svdDense" part. Since it seems like many other files removed method when it is unused. Just a minor thing.

Reply:
Good call, looks like I didn't remove unused `svdDense` methods. Fixed.